lib/watcher/heart-beat.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+exports.worker = function(handle) {
+  var _heartBeatInterval = 15 * 1000;
+  setInterval(function() {
```

Comment:
you need to keep the return value so you can implement stop, otherwise your test will never end (node doesn't exit while it has active handles, and a interal timer is a handle)